javascript-forms/form-events.js

```diff
@@ -0,0 +1,29 @@
+function handleFocus(event) {
+  console.log(event + 'was fired');
+  console.log(event.target.name);
+}
+
+function handleBlur(event) {
+  console.log(event + 'was fired');
+  console.log(event.target.name);
+}
+
+function handleInput(event) {
+  console.log(event.target.name);
+  console.log(event.target.value);
+}
+
+var userName = document.querySelector('#user-name');
+console.log(userName);
+
+var userEmail = document.querySelector('#user-email');
+console.log(userEmail);
+
+var userMessage = document.querySelector('#user-message');
+console.log(userMessage);
+
+userName.addEventListener('focus', handleFocus);
+
+userEmail.addEventListener('blur', handleBlur);
+
+userMessage.addEventListener('input', handleInput);
```

Comment:
All three event handlers should be applied to all three elements, not just one on each.

Reply:
<img width="1680" alt="Screen Shot 2022-03-30 at 12 37 42" src="https://user-images.githubusercontent.com/99709530/160917097-4f4f086a-6deb-46a4-879d-abf9a8fc4c5d.png">